packages/react-native/ReactAndroid/flipper-integration/gradle.properties

```diff
@@ -1,2 +1,2 @@
 # Version of flipper SDK to use for this integration
-FLIPPER_VERSION=0.182.0
\ No newline at end of file
+FLIPPER_VERSION=0.204.0
```

Comment:
As mentioned in the review, this needs to be split between `FLIPPER_VERSION` and `FLIPPER_FRESCO_PLUGIN_VERSION`

Reply:
Done ✅